hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/authentication.md

```diff
@@ -222,6 +226,21 @@ Note:
    configuration files MUST be in the `~/.aws/` directory on the local filesystem in
    all hosts in the cluster.
 
+### <a name="auth_simple"></a> Credentials from profile with `ProfileAWSCredentialsProvider`*
+
+This is a non-default provider that fetches credentials from a profile file,
+acting as a Hadoop wrapper around ProfileCredentialsProvider. The profile file and
+profile name are both resolved as follows.
+
+1. If the configuration setting is specified, that takes priority (`fs.s3a.auth.profile.file`
+   for profile file and `fs.s3a.auth.profile.name` for profile name).
+2. If a configuration setting is absent, but the environment variables for
+   the setting(AWS_SHARED_CREDENTIALS_FILE for profile file and AWS_PROFILE for
```

Comment:
nits
* add a space between setting and (
* use backticks to format env vars as code